pom.xml

```diff
@@ -239,11 +239,11 @@
 			<groupId>org.slf4j</groupId>
 			<artifactId>slf4j-api</artifactId>
 		</dependency>
-		<dependency>
-			<groupId>org.jresearch.dominokit</groupId>
-			<artifactId>domino-slf4j-logger</artifactId>
-			<version>${gwt-slf4j.ver}</version>
-		</dependency>
+<!--		<dependency>-->
+<!--			<groupId>org.jresearch.dominokit</groupId>-->
+<!--			<artifactId>domino-slf4j-logger</artifactId>-->
+<!--			<version>${gwt-slf4j.ver}</version>-->
+<!--		</dependency>-->
```

Comment:
I can't exclude this jar in when I import this gwt-time, for some reason my GWT build still finds it.

Reply:
Are there some issues with domino-slf4j-logger? Why do you want to exclude it?

Reply:
@foal Yes, I'm getting issues with domino classes when compiling to JavaScript. The project is quite big, so thought it would be easier to just exclude it.